hardhat.config.ts

```diff
@@ -361,6 +382,12 @@ const config: HardhatUserConfig = {
         deployment: 'usdc',
         auxiliaryBase: 'goerli'
       },
+      {
+        name: 'base',
```

Comment:
I think we're assuming that there will be a native USDC market on Base that will eventually supersede this one.

Does it make this base's name `base-usdbc`, instead of `base` (which implies that it's the primary deployment on the chain)?

```
{
  name: 'base-usdbc',
  network: 'base',
  deployment: 'usdbc',
  auxiliaryBase: 'mainnet'
}
```

Reply:
Yeah I'll change it. And I'll update the directory from `usdc` to `usdbc` as well.

Reply:
Done